osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/v0_6/impl/ChangesetManager.java

```diff
@@ -143,8 +143,7 @@ private void addChangeset(long changesetId, long userId) {
 
             // Insert the changeset tags.
             prmIndex = 1;
-            insertTagStatement.setLong(prmIndex++, changesetId);
-            insertTagStatement.setLong(prmIndex++, changesetId);
+            insertTagStatement.setLong(++prmIndex, changesetId);
```

Comment:
This causes the app to fail.  prmIndex is now set to 2 when it should be one.  It causes the following error:
org.postgresql.util.PSQLException: No value specified for parameter 1.